pkg/types/vsphere/validation/platform_test.go

```diff
@@ -57,6 +57,62 @@ func validPlatform() *vsphere.Platform {
 	}
 }
 
+func validHosts() []*vsphere.Host {
```

Comment:
```suggestion
// ValidHosts returns a slice of host configurations for use in other test suites
func ValidHosts() []*vsphere.Host {
```
 
I found a need for this in the terraform unit tests

Reply:
hmmm ... i thought I export cross package but that doesn't seem to be working for me.  for now, ignore.

Reply:
@rvanderp3 Would you like me to still make this change?